app/gui/view/component-browser/component-group/src/wide.rs

```diff
@@ -0,0 +1,432 @@
+//! A multi-column [Component Group] without header.
+//!
+//! The widget is defined by the [`View`].
+//!
+//! To learn more about component groups, see the [Component Browser Design
+//! Document](https://github.com/enso-org/design/blob/e6cffec2dd6d16688164f04a4ef0d9dff998c3e7/epics/component-browser/design.md).
+//!
+//! [Component Group]: crate::component_group::View
+
+use ensogl_core::application::traits::*;
+use ensogl_core::display::shape::*;
+use ensogl_core::prelude::*;
+
+use crate::EntryId;
+
+use enso_frp as frp;
+use ensogl_core::application::shortcut::Shortcut;
+use ensogl_core::application::Application;
+use ensogl_core::data::color::Rgba;
+use ensogl_core::display;
+use ensogl_gui_component::component;
+use ensogl_label::Label;
+use ensogl_list_view as list_view;
+use list_view::entry::AnyModelProvider;
+
+
+// =================
+// === Constants ===
+// =================
+
+const NO_ITEMS_LABEL_TEXT: &str = "No local variables";
+const ENTRY_HEIGHT: f32 = list_view::entry::HEIGHT;
+const MINIMAL_HEIGHT: f32 = ENTRY_HEIGHT;
+const COLUMNS: usize = 3;
+
+
+
+// ===============
+// === Aliases ===
+// ===============
+
+/// Type of the component group items.
+type Entry = list_view::entry::Label;
+/// An index of the column.
+type ColumnId = usize;
```

Comment:
I'd do this as a newtype instead. I like keeping such things as strongly typed things for auto-documenting purposes and a new type with auto-derives is super simple to be implemented. Also, we have a macro for that, check out `newtype_prim!`.

Reply:
Done